File: platforms/kotlin/examples/texture/mipmap/size.kt
import org.fragmentcolor.*

val pixels = ByteArray(16 * 16 * 4)
val chain = Mipmap.build(pixels, TextureFormat.RGBA8_UNORM_SRGB, Size(width=16u, height=16u, depth=null))
val tmp_size = chain.size()
val width = tmp_size.width
val height = tmp_size.height

File: platforms/python/examples/texture/mipmap/size.py
from fragmentcolor import TextureFormat, Mipmap

pixels = [0] * (16 * 16 * 4)
chain = Mipmap.build(pixels, TextureFormat.Rgba8UnormSrgb, [16, 16])
(width, height) = chain.size()
_ = (width, height)